src/components/Today.vue

```diff
@@ -58,80 +66,52 @@
         lon: "",
         res: "",
         imageWearNum: "0",
-        weather: {},
+        // weather: {},
         temp: "",
         description: "",
-        fas: "",
-        clothes,
-        WeatherIcons,
-        moment
+        fasIcon: "",
+        moment,
+        showLoading: true
       };
     },
     methods: {
       handleCloseModal() {
         this.showModal = !this.showModal;
       },
-      async getWeather(value) {
-        const response = await fetch(
-          `https://api.openweathermap.org/data/2.5/weather?lat=${
-            this.lat
-          }&lon=${this.lon}&appid=${VUE_APP_WHATHER_APP_KEY}&units=metric`
-        );
-        // this.loading = true
-        this.weather = await response.json();
-        // this.loading=false
-        console.log("json", this.weather);
-        // console.log("temp", this.weather.main.temp);
-        // console.log("weather", this.weather.weather[0].description);
-        this.temp = this.weather.main.temp;
-        this.description = this.weather.weather[0].main;
-        console.log(this.description);
-        this.fas = `fas fa-smog`;
-        this.wearIconselect();
-        this.weatherIconSelect();
-      },
-      wearIconselect() {
-        const clothesValue = Object.keys(this.clothes);
-        const stringTemp = String(parseInt(this.temp));
-        console.log(clothesValue);
-        console.log(parseInt(this.temp));
-        console.log(this.clothes[5].scope);
-        console.log(this.clothes[5].scope.indexOf(stringTemp));
-        const wearNum = clothesValue.filter(
-          e => clothes[e].scope.indexOf(stringTemp) !== -1
-        );
-        this.imageWearNum = wearNum;
-      },
-      weatherIconSelect() {
-        this.fas = WeatherIcons[this.description].iconName;
+      getWeather(lat, lon) {
+        getWeatherAPI(lat, lon)
+          .then(res => {
+            console.log("res");
+            const weather = res;
+            this.temp = parseInt(weather.main.temp);
+            this.imageWearNum = wearIconNum(this.temp);
+            this.description = weather.weather[0].main;
+            this.fasIcon = weatherIconSelet(this.description).iconName;
+          })
+          .then(() => setTimeout(() => (this.showLoading = false), 500));
+
```

Comment:
async getWeather(lat, lon) {
      const response = await getWeatherAPI(lat,lon)
      this.temp = parseInt(response.main.temp);
      this.imageWearNum = wearIconNum(this.temp)
      this.description = response.weather[0].main
      this.fasIcon = weatherIconSelet(this.description).iconName;
      setTimeout(() => (this.showLoading=false), 500)
      }